Packet++/src/LdapLayer.cpp

```diff
@@ -524,7 +562,9 @@ namespace pcpp {
 			throw std::invalid_argument("Authentication type is not sasl");
 		}
 
+		// clang-format off
 		auto authRecord = getLdapOperationAsn1Record()->getSubRecords().at(credentialIndex)->castAs<Asn1ConstructedRecord>();
+		// clang-format on
```

Comment:
We don't need to turn off clang-format here

Reply:
Fixed.